wp-content/plugins/hurumap-data/db.php

```diff
@@ -19,19 +19,15 @@ function activate_hurumap_data()
         ) $charset_collate;";
 
         $sql .= "CREATE TABLE `{$wpdb->base_prefix}flourish_charts`(
-                `id`         varchar(45) NOT NULL ,
-                `section`    varchar(45) NULL ,
-                `title`      varchar(255) NOT NULL ,
-                `subtitle`   varchar(255) NOT NULL ,
-                `file`       json NOT NULL ,
-                `published` tinyint NOT NULL DEFAULT 0,
-                `created_at` datetime NOT NULL DEFAULT NOW() ON UPDATE NOW(),
-                `updated_at` datetime NOT NULL DEFAULT NOW(),
-
-                PRIMARY KEY (`id`),
-                KEY `fkIdx_22` (`section`),
-                CONSTRAINT `FK_22` FOREIGN KEY `fkIdx_22` (`section`) REFERENCES `{$wpdb->base_prefix}chart_sections` (`id`)
-                ) $charset_collate;";
+            `id`        varchar(45) NOT NULL ,
+            `title`      varchar(255) NOT NULL ,
+            `country`   varchar(255) NOT NULL ,
+            `media_id`       int NOT NULL ,
+            `published` tinyint NOT NULL DEFAULT 0,
+            `created_at` datetime NOT NULL DEFAULT NOW() ON UPDATE NOW(),
+            `updated_at` datetime NOT NULL DEFAULT NOW(),
+            PRIMARY KEY (`id`)
+        ) $charset_collate;";
```

Comment:
So i would recommend reverting this and adding migration under a new if statement like [this](https://github.com/TakwimuAfrica/Dashboard/pull/12/files#diff-3d76cbdacfd535f4cbc8061963eeb98eR10).

so:

```
if ($version < 2) {
  ...
  update_site_option('hurumap_data_db_version', 2);
}
```